Assets/Mirror/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [29.0.1](https://github.com/MirrorNG/MirrorNG/compare/29.0.0-master...29.0.1-master) (2020-03-27)
```

Comment:
YOu should not modify the CHANGELOG yourself,

This file is autogenerated by the CI/CD pipeline.

Reply:
That wasn't something I changed. When I was trying to catch this branch up I'm pretty sure I did it wrong.

Reply:
fixed